### PR TITLE
feat: support different versions for brew install

### PR DESCRIPTION
**Pull request by kridai**
Fixes: DII-302
Test Run-
Draft release: https://github.com/kridai/homebrew-brew/actions/runs/1804599186
main release: https://github.com/kridai/homebrew-brew/actions/runs/1804479660

**Contributing to Twilio**

> All third-party contributors acknowledge that any contributions they provide will be made under the same open-source license that the open-source project is provided under.

- [x] I acknowledge that all my contributions will be made under the project's license.

### CODE REVIEW

File: .github/scripts/update-formula.sh [hunk 1]
```diff
@@ -5,6 +5,14 @@ sha=$3
 if [ "$version" == '' ] || [ "$sha" == '' ]; then
   exit
 fi
+#fetch the current version
+current_version=$(grep -w "version " Formula/$formula.rb | awk '{print $2}' | tr -d \")
+echo "Current version $current_version"
+if [ "$version" != "$current_version" ]; then
+  cp Formula/$formula.rb Formula/$formula@$current_version.rb
+  class_name=$(echo $formula | sed -e "s/\b./\u&/g")AT$(echo $current_version | tr -dc '[:alnum:]\n\r' | sed 's/[[:alpha:]]/\U&/')
```
Comment (shamantraghav) [inline .github/scripts/update-formula.sh:13]: Can you add images of this copy operation?

Comment (kridai) [reply .github/scripts/update-formula.sh:13]: Here is the image of the new formula created https://github.com/kridai/homebrew-brew/tree/main/Formula

**Review by shrutiburman — APPROVED** [review]
Good job!

**Review by charan678 — APPROVED** [review]

**Review by onuzbee — APPROVED** [review]
🚀

**Pull request merged**